include/nektar_interface/utility_sycl.hpp

```diff
@@ -0,0 +1,109 @@
+#ifndef __UTILITY_SYCL_H_
+#define __UTILITY_SYCL_H_
+
+#include <map>
+#include <memory>
+#include <neso_particles.hpp>
+#include <string>
+#include <vector>
+
+using namespace NESO::Particles;
+
+namespace NESO {
+
+/**
+ *  Get a number of local work items that should not exceed the maximum
+ *  available local memory on the device.
+ *
+ *  @param sycl_target SYCLTarget instance local memory will be allocated on.
+ *  @param num_bytes Number of bytes requested per work item.
+ *  @param default_num Default number of work items.
+ *  @returns Number of work items.
+ */
+inline size_t get_num_local_work_items(SYCLTargetSharedPtr sycl_target,
+                                       const size_t num_bytes,
```

Comment:
How is `size_t` available without specifying `std::size_t`?

Reply:
updated `size_t` to `std::size_t` across `nektar_interface`.